plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterStatus.php

```diff
@@ -122,23 +137,68 @@ protected function enqueue_data( array $stock_statuses = array() ) {
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
+		$stock_status_data    = $this->get_stock_status_counts( $block );
+		$onsale_status_data   = $this->get_onsale_status_counts( $block );
+		$featured_status_data = $this->get_featured_status_counts( $block );
```

Comment:
these methods should be called on if the corresponding status is enabled.

Reply:
Pushed up updates to address this.